.env.example

```
@@ -17,6 +17,9 @@ brawlstarsToken="123"
 twitchSecret="123"
 twitchId="123"
 prefix="t."
+sentry_dsn="https://examplePublicKey@o0.ingest.sentry.io/0"
+sentry_traces_sample_rate="0.0"
```

dotenv-linter (4.0.0)

[warning] [LowercaseKey] The sentry_traces_sample_rate key should be in uppercase

(LowercaseKey)